lib/chef/provider/package/chocolatey.rb

```diff
@@ -160,9 +160,9 @@ def choco_exe
 
         # lets us mock out an incorrect value for testing.
         def choco_install_path
-          @choco_install_path ||= powershell_out!(
+          @choco_install_path ||= powershell_exec!(
```

Comment:
Isn't this going to return a JSON object now instead of a string?

Reply:
In cases where the powershell is emitting just a string, it gets parsed as a plain string.

Reply:
Very fancy. Thanks for the clarification.

Reply:
However, looking this over more closely. It could blow up if choco is not installed and the command returns an empty string. In that case the `result` is `{}` which has no `chomp`. So I'm gonna have it check `empty?`.